content/docs/user-guide/project-structure/internal-files.md

```diff
@@ -134,7 +134,7 @@ $ cat .dvc/cache/6f/db5336fce0dbfd669f83065f107551.dir
 
 That's how DVC knows that the other two cached files belong in the directory.
 
-### Run-cache
+## Run-cache
```

Comment:
Note that this is linked to from a tooltip in https://dvc-org-exp-dvc-exps-pa-qhfdse.herokuapp.com/doc/user-guide/experiment-management#run-cache-automatic-log-of-stage-runs.